language/vm/src/vm_string.rs

```diff
@@ -107,8 +107,12 @@ pub struct VMStr(str);
 impl VMStr {
     pub fn new<'a>(s: impl AsRef<str> + 'a) -> &'a VMStr {
         let s = s.as_ref();
-        // VMStr and str have the same layout, so this is safe to do.
-        // This follows the pattern in Rust core https://doc.rust-lang.org/src/std/path.rs.html.
```

Comment:
Do you think it would be worth it to keep the reference to the `path` docs?

Reply:
I thought the docs were a  better reference. I can put it back if you think it's still useful.